public/app-icons

```diff
@@ -0,0 +1 @@
+../node_modules/@amsterdam/design-system-assets/app-icons
```

Comment:
This isn't released yet, so it's not going to work in this repo, right?

Reply:
True I guess? I don't know when we do releases. When does that happen?

Reply:
Probably this week, barring any unforseen circumstances :D 

I'll turn this PR into a draft for now